lib/scala/project-manager/src/main/scala/org/enso/projectmanager/infrastructure/languageserver/ExecutorWithUnlimitedPool.scala

```diff
@@ -108,7 +108,10 @@ object ExecutorWithUnlimitedPool extends LanguageServerExecutor {
         .flatMap(path =>
           Seq("--server-profiling-events-log-path", path.toString)
         )
+    val serverEdition =
+      Seq("--server-edition", descriptor.engineVersion.toString())
```

Comment:
Why is the option called `--server-edition`? I don't think it explains its purpose. At least I am not getting it.

Reply:
All language server related options are prefixed with the `server`. Maybe `--server-override-edition` will be more clear